refactor: use our base image to build UI's image

HazimAr opened the pull request:
closes #23 
I tested the standalone locally and everything works.

mijailr commented on Dockerfile:
```diff
@@ -1,14 +1,17 @@
+ENV NODE_ENV="production"
 
-ENTRYPOINT [ "./entrypoint.sh" ]
+ENTRYPOINT [ "./entrypoint.sh" ]
```
This overwrites the base image entrypoint

HazimAr replied:
<img width="717" alt="image" src="https://github.com/user-attachments/assets/63210258-dbc2-4622-8f0e-8a030eddd243">
It does not overwrite it as its in the `/app` directory

HazimAr commented:
@mijailr feel free to merge the PR once SSL has been tested

mijailr reviewed (approved):
Works fine on my local. Would be nice if you add how to run with SSL in the readme.

This is the command that I've used. Please modify to reflect a real usable scenario on the readme.

```shell
docker run --rm -it \
    -e SSL=enabled \
    -v ./ssl:/ssl \
    -e NEXTAUTH_URL='http://localhost:3000' \
    -e NEXTAUTH_SECRET='<any secret here>' \
    -e KEYCLOAK_HOST='http://localhost:8888' \
    -e KEYCLOAK_REALM='default' \
    -e KEYCLOAK_CLIENT_SECRET=' ' \
    -e LHUT_API_URL='http://localhost:8089' \
    -e KEYCLOAK_CLIENT_ID='123' \
    -p 3000:3000 -p 3443:3443 demo
```

mijailr commented:
Also this command to generate the local certificate.

```shell
openssl req -x509 -out tls.crt -keyout tls.key \
  -newkey rsa:2048 -nodes -sha256 \
  -subj '/CN=localhost' -extensions EXT -config <( \
   printf "[dn]\nCN=localhost\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:localhost\nkeyUsage=digitalSignature\nextendedKeyUsage=serverAuth")
```

HazimAr commented:
@sauljabin I will leave the rest of this PR to you for testing / QA, I have added a script and updated the `README` to accommodate @mijailr's request.

@sauljabin feel free to merge the PR when you are done with it.
